internal/checkers/helpers_http.go

```diff
@@ -3,12 +3,95 @@ package checkers
 import (
 	"go/ast"
 	"go/types"
+	"net/http"
+	"strconv"
 
 	"golang.org/x/tools/go/analysis"
 
 	"github.com/Antonboom/testifylint/internal/analysisutil"
 )
 
+var httpMethod = map[string]string{
+	http.MethodGet:     "http.MethodGet",
+	http.MethodHead:    "http.MethodHead",
+	http.MethodPost:    "http.MethodPost",
+	http.MethodPut:     "http.MethodPut",
+	http.MethodPatch:   "http.MethodPatch",
+	http.MethodDelete:  "http.MethodDelete",
+	http.MethodConnect: "http.MethodConnect",
+	http.MethodOptions: "http.MethodOptions",
+	http.MethodTrace:   "http.MethodTrace",
+}
+
+var httpStatusCode = map[string]string{
+	strconv.Itoa(http.StatusContinue):           "http.StatusContinue",
```

Comment:
I don't think we should consider performance for this

I would have used a `httpStatusCode = map[int]string{`

so no, strconv.Itoa here

but only when needed in the code, you can also create a dedicated testing helper if your prefer

but current code looks strange to me

Reply:
Maybe Anton will provide his guidance

Reply:
@Antonboom it's me the serial mention guy 😅😁

What are your thoughts about this?

The fact the code imported from usedstdvars is a bit strange